addons/web_editor/static/src/js/editor/odoo-editor/test/spec/editor.test.js

```diff
@@ -490,6 +490,15 @@ X[]
                         contentAfter: `<p>[]&nbsp;def</p>`,
                     });
                 });
+                it('should remove the contentEditable false element', async () => {
+                    await testEditor(BasicEditor, {
+                        contentBefore: `<p>test[]</p>\n<div contentEditable="false"><p>abc</p></div>`,
```

Comment:
Could you add some content after the `contenteditable=false` div please? So that the test can check that whatever was after it is still present at the end of the test.